app/models/spree/subscription.rb

```diff
@@ -314,5 +315,39 @@ def prior_notification_days_gap_value
           errors.add(:prior_notification_days_gap, Spree.t('subscriptions.error.should_be_earlier_than_next_delivery'))
         end
       end
+
+      def find_or_create_credit_card
+        Spree::CreditCard.where(
+          cc_type: 'visa',
+          month: 12,
+          year: 2.years.from_now.year,
+          last_digits: '1111',
+          name: 'Sean Schofield',
+          gateway_customer_profile_id: 'BGS-1234'
+        ).first_or_create!
+      end
+      
+      def find_or_initialize_payment_method
```

Comment:
i think this method has to be created via seeds or rake task and used here. like this it will be lost and our order end up missing payment method at any stage later and it can halt processing